src/mon/Monitor.cc

```diff
@@ -3940,29 +3940,35 @@ void Monitor::handle_command(MonOpRequestRef op)
     }
     f->close_section();
 
-    mgrmon()->count_metadata("ceph_version", &mgr);
-    f->open_object_section("mgr");
-    for (auto& p : mgr) {
-      f->dump_int(p.first.c_str(), p.second);
-      overall[p.first] += p.second;
+    if(!mgr.empty()){
```

Comment:
There should be space between if keyword and starting parentheses here. Also there should be space between ending parentheses and  curly bracket.